pkg/cmd/opts/install.go

```diff
@@ -344,10 +344,7 @@ func (o *CommonOptions) InstallHelm() error {
 	if err != nil || !flag {
 		return err
 	}
-	latestVersion, err := util.GetLatestVersionFromGitHub("kubernetes", "helm")
-	if err != nil {
-		return err
-	}
+	latestVersion := "v2.16.1"
```

Comment:
Ahh, I see. So this is fixing the non-Brew part. I still argue that what I said above holds, but we can take this as a stepping stone. However, the version should not be hardcoded here, but be fetched from the version stream.

Reply:
I can add an issue for that, as well as getting away from Brew.  These repairs should buy some time.

Reply:
Can we not at least use the version stream. Hard coding the version here is really not a good idea, especially since the version just got upgraded in the version stream. 

So instead of getting the latest version from GitHub, which is really the crux here, we get the latest acceptable version from the version stream. You are in `CommonOptions`, so you have access to the version stream via `o.GetVersionResolver`. All you need to do is to look up the version.

Reply:
I am fine with creating a new issue for removing Brew, since this affects really all dependencies we install and it is a bigger change. Loading the right version from the version stream, however, we should be doing here and now.

Reply:
Will do